etl/kpinetwork_analizer.py

```diff
@@ -141,8 +282,13 @@ def get_limits(limits, count):
     return index_limits
 
 
-def get_company_description(row):
-    company_id = str(uuid.uuid4())
+def get_company_description(row, env):
+    exists_id = read_ids_from_database(env, "company")
```

Comment:
Can you change the name to existing_ids?

Reply:
I have changed the name.